api/serializers.py

```
@@ -0,0 +1,68 @@
+from rest_framework import serializers
+from patients_core.models import Patient
+from diab_encounters.models import Encounter
+from diab_labs.models import LabResult
+from diab_medications.models import MedicationOrder
+from clinical_refs.models import ClinicalReference
+from ai_summarizer.models import AISummary
+
+
+class PatientSerializer(serializers.ModelSerializer):
+    primary_doctor_id = serializers.UUIDField(source='primary_doctor.id', read_only=True)
```

Ruff (0.12.2)

Line too long (89 > 88)

(E501)